plugin/plugin.go

```diff
@@ -316,10 +316,10 @@ func (p *OrmPlugin) addIncludedField(ormable *OrmableType, field *gorm.ExtraFiel
 		} else if rawType == "Inet" {
 			rawType = fmt.Sprintf("%s.Inet", p.Import(gtypesImport))
 		} else {
-			p.Fail(
-				fmt.Sprintf(
-					`Included field %q of type %q is not a recognized special type, and no package specified`,
-					field.GetName(), field.GetType()))
+			//p.Fail(
```

Comment:
for the example I wrote, having this code fail was getting in the way. I'm assuming this error was in place to give a user-friendly error for an edge case. Is there a way we can make it so that you can reference a type that's in the same package?

Reply:
This is why I said you'd probably have to put the `JoinTable` type into a separate package (then using the `package: "..."` tag. Otherwise there's this.

Reply:
do we need to fail on this? or can we add a special value or something to the `package:` option to indicate that it's from the same package? What do you think would be best?